lib/Migration/Version010200Date20220616000000.php

```diff
@@ -38,6 +38,10 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 			$table->addColumn('page_order', Types::INTEGER, [
 				'notnull' => true,
 			]);
+			$table->addColumn('settings', Types::STRING, [
```

Comment:
should not be needed, the later migration will take care

Reply:
I prefer to not have the final schema split all over the migrations, but probably just a matter of taste.